website/docs/cloud-docs/workspaces/settings/policies.mdx

```diff
@@ -0,0 +1,15 @@
+---
+page_title: Policies - Workspaces - Terraform Cloud
+description: >-
+  Learn how to adjust the policy evaluation experience according to your preferences.
+---
+
+# Policies
+
+Terraform Cloud provides two different experiences for Sentinel policy evaluations. You can adjust your settings to use the experience you prefer. By default, the newest policy evaluation experience is enabled.
```

Comment:
Do we have a link to the new policy evaluation experience? As in, **how** does this effect a user's sentinel policy experience? Can we link to the Sentinel docs somewhere or provide an example?

Suggestions for style:
```suggestion
Terraform Cloud offers two experiences for Sentinel policy evaluations. You can adjust your **Sentinel Experience** settings to your preferred experience. By default, Terraform Cloud enables the newest policy evaluation experience.
```